jobs/superkey.go

```diff
@@ -11,11 +12,13 @@ import (
 	"github.com/RedHatInsights/sources-api-go/util"
 )
 
+var _ = (Job)(&SuperkeyDestroyJob{})
```

Comment:
What is this for?

Reply:
Basically this will statically validate that the struct implements the interface at compile time. Very nice for making sure the struct continues to implement the interface since compilation will fail if a method is added or changed and the struct does not reflect it right. But I think the compiler will catch it as well - I'll remove it.

Reply:
Oooooooh. I'm okay with leaving it, but I would appreciate a "compiler check" comment or something, because I see myself forgetting about this in the future :joy: